### ISSUE TITLE
Logo ofer for JGallery

### ISSUE THREAD

**Issue by familqasimov** [junaidsshaikh/JGallery#1]
Hi sir. I would like to offer logo. If you like I can give you all the formats of the design free. If you want a change please specify.

![untitled-1](https://user-images.githubusercontent.com/36432269/42000320-5c01831c-7a71-11e8-9000-a4d8c367089b.png)

**Comment by junaidsshaikh** [comment]
Yes, I would love to get this logo. It is pretty awesome and cool @familqasimov

**Comment by familqasimov** [comment]
Can I sent Pull Reguest?
